src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -5409,52 +5230,47 @@ namespace VariableSpeedCoils {
         if (RatedWaterFlowAutoSized) {
             // FORCE BACK TO THE RATED WATER FLOW RATE WITH THE SAME RATIO DEFINED BY THE CATLOG DATA
             if (RatedCapCoolTotalAutoSized) {
-                RatedWaterVolFlowRateDes = state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).RatedCapCoolTotal *
-                                           state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).MSRatedWaterVolFlowPerRatedTotCap(NormSpeed);
+                RatedWaterVolFlowRateDes = varSpeedCoils.RatedCapCoolTotal * varSpeedCoils.MSRatedWaterVolFlowPerRatedTotCap(NormSpeed);
```

Comment:
Note that these translations are always nominal speed based, I need to get away from always using maxSpeedNum.